views/index.erb

```diff
@@ -0,0 +1,152 @@
+<style>
+  /* Split the screen in half */
```

Comment:
Great use of comments in CSS - really helpful to know how each line of code changes the appearance of the html elements.